{
	"category" : "Voyage-Mongo-Core-Exceptions",
	"classinstvars" : [
		 ],
	"classvars" : [
		 ],
	"commentStamp" : "<historical>",
	"instvars" : [
		"internalErrors" ],
	"name" : "VOMongoConnectionError",
	"pools" : [
		 ],
	"super" : "VOMongoError",
	"type" : "normal" }
